encoders/nlp/UniversalSentenceEncoder/__init__.py

```diff
@@ -16,8 +20,8 @@ class UniversalSentenceEncoder(BaseTFEncoder):
 
     def __init__(
             self,
-            model_url: str = 'https://tfhub.dev/google/universal-sentence-encoder/4',
-            *args,
+            model_url: str = UNIVERSAL_SENTENCE_ENCODER,
```

Comment:
The point would be:

to have 

```
def __init__(self, 
                    model_url: str = 'https://tfhub.dev/google/universal-sentence-encoder/4',
                    preprocessor_url: Optional[str]= None, 
                    *args, 
                   **kwargs):
....
super().__init__(*args, **kwargs)
self.preprocessor_url = preprocessor_url
self.model_url = model_url
```

and in post_init:

```
def post_init(self):
       import tensorflow_hub as hub
       preprocessor = None
       if preprocessor_url:
            preprocessor = hub.KerasLayer(preprocessor_url)
        encoder = hub.KerasLayer(model_url)
        return encoder(preprocessor(data))['default'] if preprocessor else encoder(data)['default']
```

Is that possible to work for any `universal-sentence-encoder` family?

Reply:
I think the return  line is not for this method.